src/wrapper/Plumed.h

```diff
@@ -2346,7 +2346,7 @@ class Plumed {
     char msg[__PLUMED_WRAPPER_CXX_EXCEPTION_BUFFER];
     void init(const char* msg) __PLUMED_WRAPPER_CXX_NOEXCEPT {
       this->msg[0]='\0';
-      __PLUMED_WRAPPER_STD strncat(this->msg,msg,__PLUMED_WRAPPER_CXX_EXCEPTION_BUFFER-1);
+      __PLUMED_WRAPPER_STD strncpy(this->msg,msg,__PLUMED_WRAPPER_CXX_EXCEPTION_BUFFER);
```

Comment:
since this starts with `this->msg[0]='\0';`, I think that, only in this case, strncpy should work exactly like strncat

removing the "-1" ensures that the warning is gone